.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: c
 sudo: required
 service: docker
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+install:
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+  - wget https://raw.githubusercontent.com/xapi-project/xs-opam/master/tools/xs-opam-ci.env
+  - source xs-opam-ci.env
 script: bash -ex .travis-docker.sh
 env:
   global:
-    - OCAML_VERSION="4.07"
-    - DISTRO="debian-unstable"
     - PINS="xapi-squeezed:."
     - PACKAGE="xapi-squeezed"
```

Comment:
in the example we have `PACKAGE` inside a job:

```
  jobs:
    - PACKAGE="EXAMPLE"
```

Reply:
Didn't realize we had an example (just copied one from xapi-idl), updated now, lets see if it works.

Reply:
The jobs directive is only relevant when there are multiple packages and we want to test each one separately.